code/modules/mob/living/simple_animal/friendly/snek.dm

```diff
@@ -0,0 +1,22 @@
+/mob/living/simple_animal/friendly/snek
+	name = "Snake"
+	desc = "Cute snek that needs a little boop"
+	icon_state = "Snek"
```

Comment:
It's probably a good idea to keep consistent capitalization between icon states.